<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="imagelabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>821</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>191</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>136</height>
     </rect>
    </property>
    <attribute name="label">
     <string>图像转换</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="originalButton">
        <property name="text">
         <string>初始化</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="grayButton">
        <property name="text">
         <string>转灰度图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="twoButton">
        <property name="text">
         <string>灰度图二值化</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>136</height>
     </rect>
    </property>
    <attribute name="label">
     <string>空域滤波</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>111</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="saltButton">
        <property name="text">
         <string>椒盐噪声</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="avgButton">
        <property name="text">
         <string>均值滤波</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="midButton">
        <property name="text">
         <string>中值滤波</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>136</height>
     </rect>
    </property>
    <attribute name="label">
     <string>空域图像处理</string>
    </attribute>
    <widget class="QPushButton" name="equalizeHist">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>直方图均衡化</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>136</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <attribute name="label">
     <string>图像简单处理</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="reservseButton">
        <property name="text">
         <string>图像旋转</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="antiColorButton">
        <property name="text">
         <string>图像反色(building)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scaleButton">
        <property name="text">
         <string>图像缩放(building)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>136</height>
     </rect>
    </property>
    <attribute name="label">
     <string>待添加</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>请先点我</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="fileselect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgfilelabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>731</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>图片路径不要含有中文，否则打不开</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
